src/main/java/cloud/fogbow/ras/core/plugins/interoperability/azure/securityrule/util/SecurityRuleIdContext.java

```diff
@@ -0,0 +1,36 @@
+package cloud.fogbow.ras.core.plugins.interoperability.azure.securityrule.util;
+
+import org.apache.commons.lang.StringUtils;
+
+public class SecurityRuleIdContext {
+
+    private static final int NETWORK_SECURITY_GROUP_NAME_POSITION = 0;
+    private static final int SECURITY_RULE_NAME_POSITION = 1;
+    private static final int INSTANCE_ID_ARRAY_SIZE = 2;
+    private static final String SEPARATOR = "-";
```

Comment:
I recommend changing the separator "-" to "_" since the current separator is already used in defining the resource name, if the idea is to concatenate two identifiers (_security group_ and _security rule_), the current separator can slice the identifier into four parts instead of two.

Reply:
Done.